paddle/function/ConvOpTest.h

```diff
@@ -79,45 +79,54 @@ void Convolution(const std::string& conv1,
             if (outputChannels < inputChannels) continue;
             for (size_t stride : {1, 2}) {
               for (size_t padding : {0, 1}) {
-                if (padding >= filterSize) break;
+                for (size_t dilation : {1, 3}) {
+                  if (padding >= filterSize) break;
 
-                // NNPACK only supports stride = 1 if batchSize > 1
-                if ((conv1 == "NNPACKConv-CPU" || conv2 == "NNPACKConv-CPU") &&
-                    batchSize > 1 && stride > 1)
-                  break;
+                  if ((conv1 == "NaiveConv-CPU" || conv2 == "NaiveConv-CPU") &&
+                      dilation > 1)
```

Comment:
The `NNPACKConv` also does not support dilation.

Reply:
Done